Sources/SwiftProtobufCore/BinaryDelimited.swift

```diff
@@ -44,31 +44,33 @@ public enum BinaryDelimited {
   ///   - message: The message to be written.
   ///   - to: The `OutputStream` to write the message to.  The stream is
   ///     is assumed to be ready to be written to.
+  ///   - as: The type to use for the serialization of each Message being written to the stream.
   ///   - partial: If `false` (the default), this method will check
   ///     `Message.isInitialized` before encoding to verify that all required
   ///     fields are present. If any are missing, this method throws
   ///     `BinaryEncodingError.missingRequiredFields`.
   /// - Throws: `BinaryEncodingError` if encoding fails, throws
   ///           `BinaryDelimited.Error` for some writing errors, or the
   ///           underlying `OutputStream.streamError` for a stream error.
-  public static func serialize(
+  public static func serialize<Bytes: SwiftProtobufContiguousBytes>(
     message: Message,
     to stream: OutputStream,
+    as: Bytes.Type,
     partial: Bool = false
   ) throws {
     // TODO: Revisit to avoid the extra buffering when encoding is streamed in general.
-    let serialized = try message.serializedData(partial: partial)
+    let serialized: Bytes = try message.serializedBytes(partial: partial)
```

Comment:
This is the only use of the `Bytes` generic type.  Making this method generic seems to serve no real purpose, so I would just use `[UInt8]` here and drop the generic type parameter entirely.